docs/reference/api/docker_remote_api_v1.24.md

```diff
@@ -223,6 +223,7 @@ Query Parameters:
   -   `before`=(`<container id>` or `<container name>`)
   -   `since`=(`<container id>` or `<container name>`)
   -   `volume`=(`<volume name>` or `<mount point destination>`)
+  -   `network`=(`<network name>`)
```

Comment:
actually, wondering now if we should support `<network id>` as well. Do we have that information available (i.e. would it be easy to add that?)